frontend/packages/console-app/src/components/nodes/NodeDetailsOverview.tsx

```diff
@@ -26,6 +28,7 @@ type NodeDetailsOverviewProps = {
 
 const NodeDetailsOverview: React.FC<NodeDetailsOverviewProps> = ({ node }) => {
   const machine = getNodeMachineNameAndNamespace(node);
+  const resource = node;
```

Comment:
You shouldn't need a var here. You can use `resource: node` below when you create the modal.

Reply:
updated